Controllers/Sample/SampleControllers.js

```diff
@@ -9,6 +9,50 @@ const SampleController = {
       res.status(500).json({ message: ERROR_MSGS.INTERNAL_SERVER_ERROR });
     }
   },
+  getRecipeByIngredients: async (req, res) => {
+    try {
+      const { ingredients } = req.body;
+
+      const eggplantRecipe = require("../../db/spooonacular/ingredients/eggplant.json");
+      const onionChicken = require("../../db/spooonacular/ingredients/onion,chicken.json");
+      const potatoOnionCarrot = require("../../db/spooonacular/ingredients/potato,onion,carrot.json");
+
+      let eggplant = false;
+      let onion = false;
+      let chicken = false;
+      let potato = false;
+      let carrot = false;
+
+      for (let ingredient of ingredients) {
+        if (ingredient === "eggplant") {
+          eggplant = true;
+        } else if (ingredient === "onion") {
+          onion = true;
+        } else if (ingredient === "chicken") {
+          chicken = true;
+        } else if (ingredient === "potato") {
+          potato = true;
+        } else if (ingredient === "carrot") {
+          carrot = true;
+        }
+      }
+
+      if (eggplant) {
+        res.send(eggplantRecipe);
+      } else if (onion && chicken) {
+        res.send(onionChicken);
```

Comment:
response should contain status. For code style matching, use "json" to send back data instead of "send"